hack/lib/release-images.sh

```diff
@@ -100,6 +102,11 @@ push_images() {
     docker push ${yurt_component_image}
 }
 
+kindload_images() {
+    build_images
```

Comment:
we have used `docker buildx` to replace `build_images` for building images, so you should to adapt the changes in  this pull request.

Reply:
so you can add a step in `e2e` workflow for building images by `make docker-build`